Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -190,12 +199,14 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
   }.property('FLOW.selectedControl.selectedSurveyGroup'),
 
   showDataCleaningReport: function () {
-	var opts = {}, sId = this.get('selectedSurvey');
-	FLOW.ReportLoader.load('DATA_CLEANING', sId, opts);
+  var opts = {from:this.get("reportFromDate"), to:this.get("reportToDate")};
+  var sId = this.get('selectedSurvey');
+  FLOW.ReportLoader.load('DATA_CLEANING', sId, opts);
   },
 
   showDataAnalysisReport: function () {
-	var opts = {}, sId = this.get('selectedSurvey');
+    var opts = {from:this.get("reportFromDate"), to:this.get("reportToDate")};
```

Comment:
same here regarding the code indentation

Reply:
there was no indentation issue on my side, if you notice the line that was actually added is correctly indented, used tabs